src/impl_methods.rs

```diff
@@ -1498,22 +1490,15 @@ where
             return Err(error::incompatible_shapes(&self.dim, &shape));
         }
         // Check if contiguous, if not => copy all, else just adapt strides
```

Comment:
I see this comment is entirely incorrect (copied from reshape?) but the coming change that will fix into_shape should take care of that..